package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "flow_notifications",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/main.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "build": "NODE_ENV=production webpack --progress --hide-modules --config webpack.js",
+    "dev": "NODE_ENV=development webpack --progress --config webpack.js",
+    "watch": "NODE_ENV=development webpack --progress --watch --config webpack.js",
+    "lint": "eslint --ext .js,.vue src",
+    "lint:fix": "eslint --ext .js,.vue src --fix",
+    "stylelint": "stylelint **/*.css **/*.scss **/*.vue",
+    "stylelint:fix": "stylelint **/*.css **/*.scss **/*.vue --fix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nextcloud/flow_notifications.git"
+  },
+  "author": "",
+  "license": "AGPL-3.0-or-later",
+  "bugs": {
+    "url": "https://github.com/nextcloud/flow_notifications/issues"
+  },
+  "homepage": "https://github.com/nextcloud/flow_notifications#readme",
+  "dependencies": {
+    "@nextcloud/vue": "^2.3.0",
+    "vue": "^2.6.11"
+  },
+  "devDependencies": {
+    "@nextcloud/webpack-vue-config": "^1.1.0",
+    "babel-loader": "^8.1.0",
+    "css-loader": "^3.6.0",
+    "eslint": "^6.8.0",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-loader": "^4.0.2",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
+    "node-sass": "^4.14.1",
+    "sass-loader": "^8.0.2",
+    "stylelint": "^13.6.1",
+    "stylelint-config-recommended-scss": "^4.2.0",
+    "stylelint-scss": "^3.18.0",
+    "stylelint-webpack-plugin": "^2.1.0",
+    "vue-loader": "^15.9.3",
+    "vue-template-compiler": "^2.6.11",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12",
+    "webpack-merge": "^5.1.1"
```

Comment:
This is not needed, as the default webpack config is used without modification.

Reply:
fwiw i was cherry-picking from https://github.com/nextcloud/app-tutorial 😇

Reply:
addressed